tools/cleanup/README.md

```diff
@@ -12,27 +15,44 @@ By default the current gcloud credentials are used to delete the images. If
 necessary, use the flag `--service-account _key-file.json_` to specify a service
 account that will be performing the access to the gcr.
 
-Projects to be cleaned up are expected to be defined in a `resources.yaml` file.
-To remove old images and clusters from them, call [cleanup.sh](cleanup.sh) with
-following flags:
-
-- "--project-resource-yaml" as path of `resources.yaml` file - Mandatory
-- "--re-project-name" for regex matching projects names - Optional, defaults to
-  `knative-boskos-[a-zA-Z0-9]+`
-- "--days-to-keep-images" - Optional, defaults to `365` as 1 year
-- "--hours-to-keep-clusters" - Optional, defaults to `720` as 30 days
-- "--dry-run" - Optional, performs dryrun for all gcloud functions, defaults to
+Project(s) to be cleaned up are expected to be either defined in a text
+file or passed (once or multiple times) using the `--project` flag.
+
+The following flags are available for the [cleanup.sh](cleanup.sh) script:
+
+- `--project-resource-yaml` Points to a resources file containing the names
```

Comment:
is this mutual exclusive with `--project`? expecting an instruction

Reply:
See line 18.

Reply:
The implementation actually is preferring yaml file over project, maybe we should document the behavior here?

Reply:
Why you think it's preferring?

Reply:
commented below